elsa.el

```diff
@@ -132,6 +132,19 @@
     (--each (reverse (oref (elsa-process-file file) errors))
       (princ (concat file ":" (elsa-message-format it))))))
 
+(defun elsa-run-files-and-exit ()
```

Comment:
I think maybe this could even be the default behaviour?

I'm not sure though if tools like Flymake won't complain if the return code is non-zero.  If that's the case this addition makes sense.

Reply:
This new function exits Emacs, while the old one doesn't.  Maybe that's equivalent in the way Elsa is currently used; I don't know if that's so, so I didn't want to change existing behavior.

I haven't used Elsa with Flymake or Flycheck, and I don't know how they handle the exit codes.

Reply:
OK, we can keep both of them for now.